### PR TITLE
Group job import chunks with `import_id`

### PR DESCRIPTION
**Pull request by michael-lefebvre**
add `import_id` and `part_number` to the import job API to group jobs chunk in the Dashboard.

### CODE REVIEW

File: server/lib/sync-agent.js [hunk 1]
```diff
@@ -398,7 +400,7 @@ export default class SyncAgent {
     });
   }
 
-  startImportJob(url, partNumber, size) {
+  startImportJob(url, partNumber, size, import_id) {
```
Comment (michaloo) [inline server/lib/sync-agent.js]: we could use the same approach as for `partNumber`, use camelCase as far as possible and then just set it to underscore_case in params

File: server/lib/sync-agent.js [hunk 2]
```diff
@@ -239,13 +241,13 @@ export default class SyncAgent {
   sync(stream, started_sync_at) {
     let processed = 0;
     let last_updated_at;
+    const jobId = _.get(this, "job.id", uuid());
```
Comment (michaloo) [inline server/lib/sync-agent.js]: what's the difference between jobId and importId? I assume one of them is set per chunk of imported entities and other is for the whole import.
Maybe we can put down this information in a comment?

**Comment by michael-lefebvre** [comment]
@michaloo Requested changes done.

**Comment by michaloo** [comment]
@michael-lefebvre thanks for this PR, great work.

Two notes which are not a problem in case of this codebase since it still needs some updates
 But I wanted to let you know as soon as possible:

1. in other connectors we use `const module = require("module");` instead of `import module from "module";` since NodeJS does not have nor will have a decent support for `import/export`.
2. we also tend to avoid `_.get` whenever we have flow types, since `_.get` returns `any` every time so we are loosing flow types information

**Pull request merged**